src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestHelpers/TestRunner.cs

```diff
@@ -193,25 +194,47 @@ public void AddFile(string thisFile)
 
                     test.Expected = line.Trim();
 
-                    var key = test.GetUniqueId(fileOveride);
-                    if (_keyToTests.TryGetValue(key, out var existingTest))
+                    foreach (string locale in Locales)
                     {
-                        // Must be in different sources
-                        if (existingTest.SourceFile == test.SourceFile && existingTest.SetupHandlerName == test.SetupHandlerName)
+                        Engine e = _engines.FirstOrDefault(e => e.Config.CultureInfo.Name == locale);
+
+                        if (e == null)
                         {
-                            throw ParseError(i, $"Duplicate test cases in {Path.GetFileName(test.SourceFile)} on line {test.SourceLine} and {existingTest.SourceLine}");
+                            e = new Engine(new PowerFxConfig(new CultureInfo(locale)));
```

Comment:
>e = new Engine(new PowerFxConfig(new CultureInfo(locale))); [](http://example.com/codeflow?start=28&length=59)

Double checking - for other engines (like JS, SQL) - do we need to do something differently here?